File: app/src/main/java/tigerhacks/android/tigerhacksapp/sponsors/Mentor.java
package tigerhacks.android.tigerhacksapp.sponsors;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Mentor {

    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("skills")
    @Expose
    private String skills;
    @SerializedName("contact")
    @Expose
    private String contact;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getSkills() {
        return skills;
    }

    public void setSkills(String skills) {
        this.skills = skills;
    }

    public String getContact() {
        return contact;
    }

    public void setContact(String contact) {
        this.contact = contact;
    }

}

File: app/src/main/java/tigerhacks/android/tigerhacksapp/sponsors/Mentor.kt
package tigerhacks.android.tigerhacksapp.sponsors

import android.os.Parcelable
import com.squareup.moshi.JsonClass
import kotlinx.android.parcel.Parcelize

@JsonClass(generateAdapter = true)
@Parcelize
data class Mentor(
    val sponsor: String = "",
    val name: String = "",
    val skills: String = "",
    var contact: String = ""
) : Parcelable